### PR TITLE
Example pages titles fix, alternative installs for NUC, Joule, ARTIK

### PR DESCRIPTION
**Pull request by caldav**

### CODE REVIEW

File: templates/pages/core/get-started/artik-5-10.md [hunk 1]
```diff
@@ -23,3 +25,28 @@ We will walk you through the steps of flashing Ubuntu Core on a Samsung Artik 5
...
+## Alternative install: Ubuntu Server 16.04 LTS
+
+As an alternative to Ubuntu Core, you can also install Ubuntu Server 16.04 LTS, where you can use your favourite development tools to create and run snaps.
+
+### Ubuntu image
+
+* [ARTIK 5 - Ubuntu Server 16.04 LTS image](http://people.canonical.com/~platform/snappy/artik/archive/artik5-ubuntu-server.img.tar.xz)
+* [ARTIK 10 - Ubuntu Server 16.04 LTS image](http://people.canonical.com/~platform/snappy/artik/archive/artik10-ubuntu-server.img.tar.xz)
+
+Download and copy the image for your board on an SD card by following the [installation media instructions](/core/get-started/installation-medias)
```
Comment (anthonydillon) [inline templates/pages/core/get-started/artik-5-10.md]: Need a full stop.

File: templates/pages/core/get-started/artik-5-10.md [hunk 2]
```diff
@@ -23,3 +25,28 @@ We will walk you through the steps of flashing Ubuntu Core on a Samsung Artik 5
...
+
+### Installation instructions
+
+Before installing Ubuntu, you need to set your Artik to boot from the SD card, to do so, set the "SW2" switches to `1:on` and `2:on`.
+
+Booting the board from the SD Card will start the Ubuntu installer:
+
+1. Insert the SD card in your Samsung ARTIK
+* Connect the 5V power supply to the board, then use the power button on the board to switch on your Samsung ARTIK
```
Comment (anthonydillon) [inline templates/pages/core/get-started/artik-5-10.md:47]: Should these continue to be numbered?

File: templates/pages/core/get-started/intel-joule.md [hunk 3]
```diff
@@ -42,3 +44,25 @@ We will walk you through the steps of flashing Ubuntu Core on an Intel Joule. At
...
+* [Intel Joule - Ubuntu Desktop 16.04 LTS image](http://people.canonical.com/~platform/snappy/tuchuck/tuchuck-20161014085519.img.xz)
+
+Download and copy the image on an USB flash drive by following the [installation media instructions](/core/get-started/installation-medias)
+
+### Installation instructions
+
+Booting the board from the USB flash drive will start the Ubuntu installer.
+
+1. Boot the system from the USB flash drive
+* The system will automatically execute the first stage of installation, including eMMC storage partitioning and image installation. After installation is complete, a prompt dialog will be shown and you will need to restart the system
```
Comment (anthonydillon) [inline templates/pages/core/get-started/intel-joule.md:63]: Should these continue to be numbered?

File: templates/pages/core/get-started/intel-nuc.md [hunk 4]
```diff
@@ -62,3 +64,27 @@ This step is only necessary in case you want to install Ubuntu Core on the emmc
...
+* [Intel NUC - Ubuntu Desktop 16.04 LTS image](http://people.canonical.com/~platform/snappy/nuc/ubuntu-server-16.04.1-20160817-0.iso)
+
+Download and copy the image on an USB flash drive by following the [installation media instructions](/core/get-started/installation-medias)
+
+### Installation instructions
+
+Booting the board from the USB flash drive will start the Ubuntu installer.
+
+1. Insert the USB flash drive in the NUC
+* Start the NUC and push F10 to enter the boot menu.
```
Comment (anthonydillon) [inline templates/pages/core/get-started/intel-nuc.md:83]: Should these continue to be numbered?

File: templates/pages/core/get-started/intel-nuc.md [hunk 5]
```diff
@@ -62,3 +64,27 @@ This step is only necessary in case you want to install Ubuntu Core on the emmc
...
+
+## Alternative install: Ubuntu Server 16.04 LTS
+
+As an alternative to Ubuntu Core, you can also install Ubuntu Server 16.04 LTS, where you can use your favourite development tools to create and run snaps.
+
+### Ubuntu image
+
+* [Intel NUC - Ubuntu Desktop 16.04 LTS image](http://people.canonical.com/~platform/snappy/nuc/ubuntu-server-16.04.1-20160817-0.iso)
+
+Download and copy the image on an USB flash drive by following the [installation media instructions](/core/get-started/installation-medias)
```
Comment (anthonydillon) [inline templates/pages/core/get-started/intel-nuc.md]: Missing a full stop

File: templates/pages/core/get-started/intel-joule.md [hunk 6]
```diff
@@ -42,3 +44,25 @@ We will walk you through the steps of flashing Ubuntu Core on an Intel Joule. At
...
+
+## Alternative install: Ubuntu Desktop 16.04 LTS
+
+As an alternative to Ubuntu Core, you can also install Ubuntu Desktop 16.04 LTS, where you can use your favourite development tools to create and run snaps.
+
+### Ubuntu image
+
+* [Intel Joule - Ubuntu Desktop 16.04 LTS image](http://people.canonical.com/~platform/snappy/tuchuck/tuchuck-20161014085519.img.xz)
+
+Download and copy the image on an USB flash drive by following the [installation media instructions](/core/get-started/installation-medias)
```
Comment (anthonydillon) [inline templates/pages/core/get-started/intel-joule.md]: Missing a full stop.

**Review by anthonydillon — CHANGES_REQUESTED** [review]
Thanks @caldav. Looks good. Missing a few full stops and a couple of questions.

File: templates/pages/core/get-started/intel-nuc.md (hunk 4 above)
Comment (caldav) [reply templates/pages/core/get-started/intel-nuc.md:83]: No need, Markdown interprets it as a numbered list as long as it starts with a 1.

File: templates/pages/core/get-started/intel-joule.md (hunk 3 above)
Comment (caldav) [reply templates/pages/core/get-started/intel-joule.md:63]: No need, Markdown interprets it as a numbered list as long as it starts with a 1.

File: templates/pages/core/get-started/artik-5-10.md (hunk 2 above)
Comment (caldav) [reply templates/pages/core/get-started/artik-5-10.md:47]: No need, Markdown interprets it as a numbered list as long as it starts with a 1.

**Review by anthonydillon — APPROVED** [review]
Good stuff :+1: thanks.

**Pull request merged**